src/conditionals/wpml-conditional.php

```diff
@@ -0,0 +1,17 @@
+<?php namespace Yoast\WP\Free\Conditionals;
+
+/**
+ * Class WPML_Conditional
+ * @package Yoast\WP\Free\Conditionals
+ */
+class WPML_Conditional implements Conditional {
+
+	/**
+	 * Returns whether or not this conditional is met.
+	 *
+	 * @return boolean Whether or not the conditional is met.
+	 */
+	public function is_met() {
+		return function_exists( 'icl_object_id' );
```

Comment:
Is there a WPML constant you can check for instead?

Reply:
Based on their own answers on their forums, this seems to be their 'best practice' answer.